src/overmap.cpp

```diff
@@ -2988,6 +2988,21 @@ bool overmap::check_ot_subtype( const std::string &otype, int x, int y, int z )
     return is_ot_subtype( otype.c_str(), oter );
 }
 
+bool overmap::check_overmap_special_type( const overmap_special_id &id, int x, int y, int z ) const
```

Comment:
Done.

When updating the [not-yet-PR'd changes](https://github.com/CleverRaven/Cataclysm-DDA/compare/master...ralreegorganon:finding-overmap-special) that depend on this, I remembered the other reason I kept the API homogeneous--the overmapbuffer versions of those two functions call the corresponding versions of get_om_global that take the coordinates by reference and adjust them to be overmap local, and return the overmap, while the tripoint version of get_om_global doesn't currently return adjusted coordinates. 

It's the sort of thing that it would actually probably be better (or at least less surprising) for the get_om_global functions to return a new type that has both the overmap and the adjusted coordinates, rather than changing coordinates passed by reference for the xyz version and not for the tripoint.